planet/src/main/webapp/js/itinerary.js

```diff
@@ -26,9 +26,23 @@ window.handleListOptionChange = handleListOptionChange;
 window.addEvent = addEvent;
 window.saveEvents = saveEvents;
 window.generateItinerary = generateItinerary;
+window.initAutocomplete = initAutocomplete; 
 
 const database = firebase.database();
 
+/** Adds autocomplete to input boxes */
+function initAutocomplete() {
+    let startAddress = document.getElementById('starting-address');
+    let eventAddress = document.getElementById('add-event-address');
+
+    let options = {
+        types: ['geocode']
+    };
+
+    new google.maps.places.Autocomplete(startAddress,options); 
```

Comment:
Consider assigning these to variables or globals so that they can be referenced in the future if needed